docs/source/azure/gettingstarted.rst

```diff
@@ -11,25 +11,43 @@ Getting Started
 Install Cloud Custodian and Azure Plugin
 ----------------------------------------
 
-The Azure provider must be installed as a separate package in addition to c7n.
+Cloud Custodian is a Python application and supports Python 2 and 3 on Linux and Windows.
+We recommend using Python 3.6 or higher.
+
+The Azure provider is an additional package which is installed in addition to c7n.
+
+
+### Option 1: Install released packages to local Python Environment
 
 .. code-block:: bash
 
-  $ virtualenv custodian
-  $ source custodian/bin/activate
-  (custodian) $ pip install c7n
-  (custodian) $ pip install c7n_azure
+$ pip install c7n
+$ pip install c7n_azure
 
 
-If you prefer to install the latest from source control you can do so as follows:
+### Option 2: Install latest from the repository
 
 .. code-block:: bash
 
-  $ git clone https://github.com/capitalone/cloud-custodian.git
-  $ virtualenv custodian
-  $ source custodian/bin/activate
-  (custodian) $ pip install ./cloud-custodian
-  (custodian) $ pip install ./cloud-custodian/tools/c7n_azure
+$ git clone https://github.com/capitalone/cloud-custodian.git
+$ cd cloud-custodian
+$ pip install -e ./cloud-custodian
+$ pip install -e ./cloud-custodian/tools/c7n_azure
+
+
+### Option 3: Run from docker image
+
+Note: Azure CLI auth will require additional workarounds in this scenario.
+
+.. code-block:: bash
+
+$ docker pull cloudcustodian/c7n
+$ docker run -it
+    -v $(pwd)/output:/output
+    -v $(pwd)/policy.yml:/policy.yml
+    --env-file <(env | grep "^AZURE")
```

Comment:
I think we should add `export AZURE_CLIENT_ID` etc here if we use `env | grep ^AZURE`